# I. Basics - biology and atabases

* [Encode Project](https://www.encodeproject.org/)
* [Downloads and more](https://application.wiley-vch.de/HOME/bioinformatik/index.html)

## 1. Basics in biology

### 1.1 DNA

* [DNA](https://en.wikipedia.org/wiki/DNA)

### 1.2 Genetic code and gencomposition

* [Genetic code](https://en.wikipedia.org/wiki/Genetic_code)

### 1.3 Transcriptions

* [Transcription (biology)](https://en.wikipedia.org/wiki/Transcription_(biology))

### 1.4 RNA

* [RNA](https://en.wikipedia.org/wiki/RNA)

### 1.5 Proteins 

* [Protein](https://en.wikipedia.org/wiki/Protein)

### 1.6 Peptide bonding

* [Peptide bond](https://de.wikipedia.org/wiki/Peptidbindung)

### 1.7 Conformation of amino acid side chains

* [Amino acid](https://en.wikipedia.org/wiki/Amino_acid)
* [Side chain](https://en.wikipedia.org/wiki/Side_chain)

### 1.8 Ramachandran plot

* [Ramachandran plot](https://en.wikipedia.org/wiki/Ramachandran_plot)

### 1.9 Hierarchical description of protein structures

* [Protein structure](https://en.wikipedia.org/wiki/Protein_structure)

### 1.10 Secondary structural elements

* [Protein secondary structure](https://en.wikipedia.org/wiki/Protein_secondary_structure)

### 1.11 Alpha-helix

* [Alpha helix](https://en.wikipedia.org/wiki/Alpha_helix)

### 1.12 Beta sheet

* [Beta sheet](https://en.wikipedia.org/wiki/Beta_sheet)

### 1.13 super secondary structure elements

* [Super secondary structures](https://www.spektrum.de/lexikon/biochemie/supersekundaerstrukturen/6022)

### 1.14 Protein domains

* [Protein domain](https://en.wikipedia.org/wiki/Protein_domain)

### 1.15 Proteinfamilies

* [Proteinfamily](https://en.wikipedia.org/wiki/Protein_family)

### 1.16 Enzyme

* [Enzyme](https://en.wikipedia.org/wiki/Enzyme)

### 1.17 Protein complex

* [Protein complex](https://en.wikipedia.org/wiki/Protein_complex)

### 1.18 Evolutionaray processes

* [Evolutionary processes](https://de.wikipedia.org/wiki/Evolution#:~:text=Diese%20Prozesse%20sind%20die%20zentrale,%2C%20Rekombination%2C%20Selektion%20und%20Gendrift.)

### 1.19 Technical terms

 * [Homology](https://www.bionity.com/de/lexikon/Homologie_%28Biologie%29.html)
 * [Genotype](https://en.wikipedia.org/wiki/Genotype)
 * [Phenotype](https://en.wikipedia.org/wiki/Phenotype)
 * [Prokaryote](https://en.wikipedia.org/wiki/Prokaryote)
 * [Eukaryote](https://en.wikipedia.org/wiki/Eukaryote)
 * [Nomenklatur (biologie)](https://de.wikipedia.org/wiki/Nomenklatur_(Biologie))
 * [Microorganism](https://en.wikipedia.org/wiki/Microorganism)
 * [Gram stain](https://en.wikipedia.org/wiki/Gram_stain)
 * [Genome](https://en.wikipedia.org/wiki/Genom)
 * [Metagenomics](https://en.wikipedia.org/wiki/Metagenomics)
 * [Systembiologie](https://en.wikipedia.org/wiki/Systems_biology)
 * [Genomics](https://en.wikipedia.org/wiki/Genomics)
 * [Transcriptome](https://en.wikipedia.org/wiki/transcriptome)
 * [Proteomik](https://de.wikipedia.org/wiki/Proteomik)
 * [Metabolomics](https://en.wikipedia.org/wiki/Metabolomics)
 * [in vitro](https://en.wikipedia.org/wiki/In_vitro)
 * [in vivo](https://en.wikipedia.org/wiki/In_vivo)
                      
## 2. Sequences and their functions

### 2.1 Definitions and operators

* [DNA sequencing](https://en.wikipedia.org/wiki/DNA_sequencing)
* [RNA sequencing](https://en.wikipedia.org/wiki/Sequencing#RNA_sequencing)
* [Protein sequencing](https://en.wikipedia.org/wiki/Protein_sequencing)

#### Notation

![Sequences in a mathematical sense](images/sequences.jpg)
   
#### Examples for alphabets

* [American Standard Code for Information Interchange](https://en.wikipedia.org/wiki/ASCII)
* [Genetischer Code/Alphabet/ACGU/Beispiel](https://de.wikiversity.org/wiki/Genetischer_Code/Alphabet/ACGU/Beispiel)
* [Amino acids](https://www.chemeurope.com/en/encyclopedia/Amino_acid.html)
* [Nucleobases](https://en.wikipedia.org/wiki/Nucleobase)

##### Alphabet for DNA sequences

![Alphabet for DNA sequences](images/alphabet_dna_sequences.jpg)

##### Alphabet for protein sequences

![Alphabet for protein sequences](images/alphabet_protein_sequences.jpg)

##### Alternative amino acid sequencies

![Alternative alphabet for amino acids](images/alternative_alphabet.jpg)

### 2.2 DNA sequencies

* [DNA sequencing](https://en.wikipedia.org/wiki/DNA_sequencing)

### 2.3 Protein sequencies

* [Protein sequencing](https://en.wikipedia.org/wiki/Protein_sequencing)
   * [FASTA format](https://en.wikipedia.org/wiki/FASTA_format)
   
Proteinsequence of CAP-Monomers in FASTA file format:

![CAP-Monomer in fasta file format](images/cap_monomer_fasta_format.jpg)

 * [multiple-fasta-file](https://www.metagenomics.wiki/tools/fastq/multi-fasta-format)
   
### 2.4 Compare sequence compositions

![Calculate amino acid specific diffs and sum them up](images/calc_amino_diff.jpg)

#### Comparison of codon frequencies

![Compare codon frequencies formula](images/codon_freq.jpg)

![Codon usage contrast formula](images/codon_usage_contrast.jpg)

#### Compositional analysis and metagenome projects

* In which applications is a comparison of the composition of sequences of interest:
   * Optimization of heterogenic expressions
   * [Metagenome projects](https://de.wikipedia.org/wiki/Metagenomik)

### 2.5 Ontologies

* [Ontology]([https://de.wikipedia.org/wiki/Ontologie](https://en.wikipedia.org/wiki/Ontology))

* [Ontologies in biology: design, applications and future challenges](https://www.nature.com/articles/nrg1295)

* [Enzyme Commission number](https://en.wikipedia.org/wiki/Enzyme_Commission_number)

#### Ontology: Terms and their relationships

* [Ontology]([https://en.wikipedia.org/wiki/Ontology](https://warwick.ac.uk/fac/soc/ces/research/current/socialtheory/maps/ology/#:~:text=In%20brief%2C%20ontology%2C%20as%20a,what%20you%20set%20to%20examine.))

* [Ontology (computer science)](https://en.wikipedia.org/wiki/Ontology_(computer_science))

#### Controlled Vocabulary

* [2. What Are Controlled Vocabularies?](https://www.getty.edu/research/publications/electronic_publications/intro_controlled_vocab/what.pdf)

* [Transitive Relation](https://de.wikipedia.org/wiki/Transitive_Relation)

#### An ontology to describe genetic products

* [Gene Ontology](https://de.wikipedia.org/wiki/Gene_Ontology)

* [The Protein Feature Ontology: A Tool for the Unification of Protein Annotations](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2912506/)

* [Systems Biology Ontology](https://de.wikipedia.org/wiki/Systems_Biology_Ontology)
  * [https://www.ebi.ac.uk/ols/ontologies/sbo](https://www.ebi.ac.uk/ols/ontologies/sbo)

#### Topology of the GO Ontology

* [Topology](https://en.wikipedia.org/wiki/Topology)

### 2.6 Analysis fo the enrichment of GO terms

* [GO enrichment analysis](http://geneontology.org/docs/go-enrichment-analysis/#:~:text=One%20of%20the%20main%20uses,annotations%20for%20that%20gene%20set.)

### 2.7 Semantic similarity of GO terms

* [Semantic Similarity in the Gene Ontology](https://link.springer.com/protocol/10.1007/978-1-4939-3743-1_12)

#### 2.7.1 Evaluation with information-theoretical approaches

p 48ff.

#### 2.7.2 Compare with a graph theoretical method

p 50ff.

## 3. Databases

* [Basic Local Alignment Search Tool](https://blast.ncbi.nlm.nih.gov/Blast.cgi)
  * [Download BLAST Software and Databases](https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download)
  * [BLAST-Algorithmus](https://www.wiki-data.de-de.nina.az/BLAST-Algorithmus.html)

##### Integrated database systems

* [GQuery (USA)](https://library.medicine.yale.edu/find/title/entrez)

* [UniProt](https://www.uniprot.org/)

* [EMBL-EBI (Europe)](https://www.ebi.ac.uk/)
  * [Flat File Example](https://ena-docs.readthedocs.io/en/latest/submit/fileprep/flat-file-example.html#:~:text=The%20EMBL%20flat%20file%20format,preparing%20flat%20files%20for%20submission.)

### 3.1 Nucleotide sequence databases

* [ENA](https://www.ebi.ac.uk/ena/browser/home)

* [GenBank](https://www.ncbi.nlm.nih.gov/genbank/)

### 3.2 RNA Sequence databases

* [ENCODE Project](https://www.encodeproject.org/)

* [Rfam (most imprtant database)](https://rfam.org/) 

### 3.3 Proteinsequenzdatenbanken

* [UniProt (most important database)](https://www.uniprot.org/)

### 3.4 3d-structure of databases

* [most important databse for protein-3-d-structures](https://www.rcsb.org/)

#### Nuclein acid databases

* [Nuclein Acid Database](http://ndbserver.rutgers.edu/)

#### Cryo-EM-Database

* [EMDB](https://www.ebi.ac.uk/emdb/)

### 3.5 SMART: Analysis fo domain architecture

* [SMART](http://smart.embl-heidelberg.de/)

### 3.6 STRING: Proteins and their interactions

* [STRING database](https://string-db.org/)

### 3.7 Structural Classification of proteins

* [SCOP 2](https://scop.mrc-lmb.cam.ac.uk/)

### 3.8 Pfam: Compilation of protein families

* []()

### 3.9 COG and eggNOG: Group o forthological Genes

* [COG](https://www.ncbi.nlm.nih.gov/research/cog-project/)

* [eggNOG](http://eggnog5.embl.de/#/app/home)

### 3.10 KEGG: Gene, Genome and illnesses

* [KEGG](https://www.genome.jp/kegg/)

### 3.11 NCBI databases: literature und biological know-how

* [NCBI database]([https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3531099/#:~:text=NCBI%20resources%20include%20Entrez%2C%20the,Genome%20and%20related%20tools%2C%20the](https://www.ncbi.nlm.nih.gov/guide/data-software/))

### 3.12 More databases

* [http://www.ebi.ac.uk/thornton-srv/databases/pdbsum/](http://www.ebi.ac.uk/thornton-srv/databases/pdbsum/)
* [https://pubmed.ncbi.nlm.nih.gov/](https://pubmed.ncbi.nlm.nih.gov/)
* [https://www.ncbi.nlm.nih.gov/pmc?db=PMC](https://www.ncbi.nlm.nih.gov/pmc?db=PMC)
* [https://www.ncbi.nlm.nih.gov/snp/](https://www.ncbi.nlm.nih.gov/snp/)
* [https://www.ncbi.nlm.nih.gov/nucleotide/](https://www.ncbi.nlm.nih.gov/nucleotide/)
* [https://www.ncbi.nlm.nih.gov/taxonomy](https://www.ncbi.nlm.nih.gov/taxonomy)
* [https://www.genome.gov/Funded-Programs-Projects/NHGRI-Genome-Sequencing-Program](https://www.genome.gov/Funded-Programs-Projects/NHGRI-Genome-Sequencing-Program)
* [https://www.ncbi.nlm.nih.gov/geoprofiles/](https://www.ncbi.nlm.nih.gov/geoprofiles/)
* [https://www.ncbi.nlm.nih.gov/gds](https://www.ncbi.nlm.nih.gov/gds)
* [https://www.rcsb.org/](https://www.rcsb.org/)
* [https://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi](https://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi)
* [https://www.ncbi.nlm.nih.gov/Structure/biosystems/docs/biosystems_about.html](https://www.ncbi.nlm.nih.gov/Structure/biosystems/docs/biosystems_about.html)
* [https://www.ebi.ac.uk/interpro/](https://www.ebi.ac.uk/interpro/)
* [https://www.brenda-enzymes.org/](https://www.brenda-enzymes.org/)
* [https://mibig.secondarymetabolites.org/](https://mibig.secondarymetabolites.org/)
* [https://biocyc.org/](https://biocyc.org/)
* [https://genexplain.com/transfac/](https://genexplain.com/transfac/)
* [https://www.yeastgenome.org/](https://www.yeastgenome.org/)
* [http://119.3.41.228/atmad/index.php](http://119.3.41.228/atmad/index.php)
* [https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-016-0276-z](https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-016-0276-z)
* [https://reactome.org/](https://reactome.org/)

# II. Learn, Optimize and Decide

* [Stochastic](https://en.wikipedia.org/wiki/Stochastic)
* [Bayes' theorem](https://en.wikipedia.org/wiki/Bayes%27_theorem)
* [Unsupervised learning](https://en.wikipedia.org/wiki/Unsupervised_learning)
  * [Cluster analysis](https://en.wikipedia.org/wiki/Cluster_analysis)
* [Artificial neural network](https://en.wikipedia.org/wiki/Artificial_neural_network)
* [Genetic algorithm](https://en.wikipedia.org/wiki/Genetic_algorithm)

## 4. Basic concepts of stochastics

* [Statistics](https://en.wikipedia.org/wiki/Statistics)
* [Probability theory](https://en.wikipedia.org/wiki/Probability_theory)
* [Experiment (probability theory)](https://en.wikipedia.org/wiki/Experiment_(probability_theory))
* [Statistical_hypothesis_testing](https://en.wikipedia.org/wiki/Statistical_hypothesis_testing)
* [Model](https://en.wikipedia.org/wiki/Model)

### 4.1 Basic terms of descriptive statistics
* [Descriptive statistics](https://en.wikipedia.org/wiki/Descriptive_statistics)
  * [absolute frequencies](https://peterstatistics.com/Ultimate/Session2/S02E01-AbsFreq-RelFreq/S02E01-TablesNominal.html#:~:text=An%20alternative%20method%20would%20be,results%20in%204%20%3D%2040%25.)
  * [relative frequencies](https://peterstatistics.com/Ultimate/Session2/S02E01-AbsFreq-RelFreq/S02E01-TablesNominal.html#:~:text=An%20alternative%20method%20would%20be,results%20in%204%20%3D%2040%25.)
  * [additive smoothing](https://www.youtube.com/watch?v=wqxlM591dBY)
 
### 4.2 Random variable, probability measure

* [discrete random variable](https://en.wikipedia.org/wiki/Random_variable)
* [Cumulative distribution function](https://en.wikipedia.org/wiki/Cumulative_distribution_function)
* [Probability mass function](https://en.wikipedia.org/wiki/Probability_mass_function)

### 4.3 Urn experiments and discrete distributions

#### Binomial distribution as a model for independend experiments

* [Binomial distribution](https://en.wikipedia.org/wiki/Binomial_distribution)

#### Hypergeometric distribution

* [Hypergeometric distribution](https://en.wikipedia.org/wiki/Hypergeometric_distribution)

#### Geometric distribution

* [Geometric distribution](https://en.wikipedia.org/wiki/Geometric_distribution)

### 4.4 Axioms of Kolmogorov

* [Axioms of Kolmogorov](https://de.wikipedia.org/wiki/Wahrscheinlichkeitstheorie#Axiome_von_Kolmogorow)

### 4.5 Conditional probability, independence, Bayes' theorem

* [Conditional independence]([https://de.wikipedia.org/wiki/Stochastische_Unabh%C3%A4ngigkeit](https://en.wikipedia.org/wiki/Conditional_independence))
* [Bayes' theorem](https://en.wikipedia.org/wiki/Bayes%27_theorem)

### 4.6 Markov chains

* [https://en.wikipedia.org/wiki/Markov_chain](https://en.wikipedia.org/wiki/Markov_chain)

### 4.7 Expected value, variance

* [Expected value](https://en.wikipedia.org/wiki/Expected_value)
* [Variance](https://en.wikipedia.org/wiki/Variance)

## 4.8 Important probability distributions 

### 4.8.1 [Discrete probability distribution](https://de.wikipedia.org/wiki/Diskrete_Wahrscheinlichkeitsverteilung)

### 4.8.2 [Total continuous distribution](https://de.wikipedia.org/wiki/Stetige_Wahrscheinlichkeitsverteilung)

* [Univariate distribution](https://en.wikipedia.org/wiki/Univariate_distribution)
* [Z-Score](https://en.wikipedia.org/wiki/Standard_score)
* [Exponential distribution](https://en.wikipedia.org/wiki/Exponential_distribution)

### 4.9 [Predict random variables](https://de.wikipedia.org/wiki/Stichprobenvarianz_(Sch%C3%A4tzfunktion))

* [Likelihood function](https://en.wikipedia.org/wiki/Likelihood_function)
* [Maximum likelihood estimation](https://en.wikipedia.org/wiki/Maximum_likelihood_estimation)

### 4.10 Basics of statistical tests

* [Multiple comparisons problem](https://en.wikipedia.org/wiki/Multiple_comparisons_problem)

### 4.11 [Neyman-Pearson method](https://en.wikipedia.org/wiki/Neyman%E2%80%93Pearson_lemma)

## 5. Bayesian decision theory and classifiers

### 5.1 [Bayesian decision theory](https://en.wikipedia.org/wiki/Bayes_estimator)

#### 5.1.1 Examples

* [https://scikit-learn.org/stable/modules/naive_bayes.html](https://scikit-learn.org/stable/modules/naive_bayes.html)

### 5.2 [Marginalize](https://de.wikibooks.org/wiki/Formelsammlung_Statistik/_Wahrscheinlichkeitsrechnung)

### 5.3 [Boosting](https://en.wikipedia.org/wiki/Boosting_(machine_learning))

### 5.4 [ROC (Receiver operating characteristic) curve](https://en.wikipedia.org/wiki/Receiver_operating_characteristic)

### 5.5 [Statistical methods for small training sets](https://measuringu.com/small-n/)

## 6. Classical cluster and classification methods

##### [Cluster analysis](https://en.wikipedia.org/wiki/Cluster_analysis)

### 6.1 [Metrics and cluster analysis](https://en.wikipedia.org/wiki/Cluster_analysis)

### 6.2 Mean squared error

* [Mean squared error](https://en.wikipedia.org/wiki/Mean_squared_error)

### 6.4 [k-means clustering](https://en.wikipedia.org/wiki/K-means_clustering)

### 6.5 [Hierarchical clustering](https://en.wikipedia.org/wiki/Hierarchical_clustering)

### 6.6 [Affinity propagation](https://en.wikipedia.org/wiki/Affinity_propagation)

### 6.7 [Evaluation of cluster analysis](https://en.wikipedia.org/wiki/Cluster_analysis)

### 6.8 [Overlapping clusters](https://www.ibm.com/docs/de/ibm-mq/8.0?topic=clusters-overlapping)

### 6.9 [k-nearest neighbors algorithm](https://en.wikipedia.org/wiki/K-nearest_neighbors_algorithm)

## 7. [Neural network](https://en.wikipedia.org/wiki/Artificial_neural_network)

### 7.1 [Neural network architectures](https://towardsdatascience.com/neural-network-architectures-156e5bad51ba)

### 7.2 [Perceptron](https://en.wikipedia.org/wiki/Perceptron)

* [Treshold functions](https://de-academic.com/dic.nsf/dewiki/1262334)

### 7.3 [Boolean functions](https://en.wikipedia.org/wiki/Boolean_function)

### 7.4 Solvability of classification tasks

### 7.6 [Lernen in neuronalen Netzen](https://de.wikipedia.org/wiki/Deep_Learning)

### 7.7 [Backpropagation](https://en.wikipedia.org/wiki/Backpropagation)

### 7.8 Coding the input

* [https://www.askpython.com/python/examples/backpropagation-in-python](https://www.askpython.com/python/examples/backpropagation-in-python)

### 7.9 [Self-organising cards](https://medium.com/machine-learning-researcher/self-organizing-map-som-c296561e2117)

### 7.10 [Deep architectures](https://developer.ibm.com/articles/cc-machine-learning-deep-learning-architectures/)

### 7.11 A neurnal network

* [https://realpython.com/python-ai-neural-network/](https://realpython.com/python-ai-neural-network/)
* [https://www.infoworld.com/article/3685569/how-to-build-a-neural-network-in-java.html](https://www.infoworld.com/article/3685569/how-to-build-a-neural-network-in-java.html)

### 7.12 [Neocognitron](https://en.wikipedia.org/wiki/Neocognitron)

### 7.13 [Convolutional Neural Network](https://de.wikipedia.org/wiki/Convolutional_Neural_Network)

### 7.14 Long-term storage of input data

##### [Recurrent neural networks](https://en.wikipedia.org/wiki/Recurrent_neural_network)

##### [Long short-term memory](https://en.wikipedia.org/wiki/Long_short-term_memory)

### 7.15 [Attention based networks](https://en.wikipedia.org/wiki/Attention_(machine_learning)#:~:text=In%20artificial%20neural%20networks%2C%20attention,important%2C%20parts%20of%20the%20data.)

## 8. [Genetic algorithms](https://en.wikipedia.org/wiki/Genetic_algorithm)

### 8.7 [Genetisches Programmieren](https://en.wikipedia.org/wiki/Genetic_programming)

# III. Bioinformatics algorithms and models

## 9. [Pairwise sequence alignment](https://en.wikipedia.org/wiki/Sequence_alignment)

### 9.1 [Dotplots](https://en.wikipedia.org/wiki/Dot_plot_(bioinformatics))

### 9.2 [Alignment procedure](https://en.wikipedia.org/wiki/Sequence_alignment)

### 9.3 [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance)

### 9.4 Determine the similarity of the sequences

* [https://en.wikipedia.org/wiki/Sequence_alignment](https://en.wikipedia.org/wiki/Sequence_alignment)

### 9.5 [Optimales Bewerten von Lücken](https://www.techtarget.com/searchcio/definition/gap-analysis)

## 10. [Sequenzmotive](https://en.wikipedia.org/wiki/Sequence_motif)

### 10.1 [Signature](https://en.wikipedia.org/wiki/Gene_signature)

### 10.2 [PROSITE database](https://en.wikipedia.org/wiki/PROSITE)

### 10.3 [BLOCKS database](http://blocks.fhcrc.org/)

### 10.4 [Sequence profile](https://en.wikipedia.org/wiki/Sequence_profiling_tool)

### 10.5 [Promoter (genetics)](https://en.wikipedia.org/wiki/Promoter_(genetics))

### 10.6 Possibilities and limits of profile-based classification

### 10.7 [Sequence logo](https://en.wikipedia.org/wiki/Sequence_logo)

### 10.8 [Consensus sequence](https://en.wikipedia.org/wiki/Consensus_sequence)

### 10.9 [Low complexity sequences](https://en.wikipedia.org/wiki/Low_complexity_regions_in_proteins)

### 10.10 [SEG algorithm](https://bioinformaticshome.com/tools/cnv/descriptions/SEG.html#gsc.tab=0)

## 11. [Scoring-Schemata](https://application.wiley-vch.de/HOME/bioinformatik/scoring/ueb/scor_ueb.htm)

### 11.1 [Theorie of scoring-matrices](https://application.wiley-vch.de/HOME/bioinformatik/scoring/ueb/scor_ueb.htm)

### 11.3 [Identity matrix](https://en.wikipedia.org/wiki/Identity_matrix)

### 11.4/5 [Point accepted mutation](https://en.wikipedia.org/wiki/Point_accepted_mutation)

### 11.6 [Alternative to PAM: the JTT-Matrix](https://www.researchgate.net/profile/Tuan-Nguyen-128/publication/311487560_Transcriptomic_characterization_and_curation_of_candidate_neuropeptides_regulating_reproduction_in_the_eyestalk_ganglia_of_the_Australian_crayfish_Cherax_quadricarinatus/links/5848a03708ae61f75de3567c/Transcriptomic-characterization-and-curation-of-candidate-neuropeptides-regulating-reproduction-in-the-eyestalk-ganglia-of-the-Australian-crayfish-Cherax-quadricarinatus.pdf)

### 11.7 [BLOSUM Matrices](https://en.wikipedia.org/wiki/BLOSUM)

### 11.8 [Matrix entropy](https://en.wikipedia.org/wiki/Entropy_(information_theory))

### 11.9 [Scoring schemata and applications](https://application.wiley-vch.de/HOME/bioinformatik/scoring/ueb/scor_ueb.htm)

### 11.10 [Scoring-Functions](https://en.wikipedia.org/wiki/Scoring_functions_for_docking#:~:text=Scoring%20functions%20are%20widely%20used,starting%20points%20for%20drug%20discovery)

## 12. FASTA and the BLAST-Suite

### 12.1 FASTA

* [FASTA Sequence Comparison at the U. of Virginia](https://fasta.bioch.virginia.edu/fasta_www2/fasta_list2.shtml)

* [Software github package](https://github.com/wrpearson/fasta36/tree/v36.3.8)

* [FASTA](https://www.ebi.ac.uk/Tools/sss/fasta/)

### 12.2 BLAST

* [Basic Local Alignment Search Tool](https://blast.ncbi.nlm.nih.gov/Blast.cgi)

### 12.4 Approaches to improving performance

### 12.5 [Profile based sequence comparison](https://pubmed.ncbi.nlm.nih.gov/10716175/)

### 12.6 [PSI-BLAST](https://en.wikipedia.org/wiki/CS-BLAST)

* [https://application.wiley-vch.de/HOME/bioinformatik/fastblast/ueb/beybla_ueb.htm#:~:text=PSI%2DBLAST%20ist%20empfindlicher%20als,der%20mit%20einer%20Iterationsrunde%20auskommt.](https://application.wiley-vch.de/HOME/bioinformatik/fastblast/ueb/beybla_ueb.htm#:~:text=PSI%2DBLAST%20ist%20empfindlicher%20als,der%20mit%20einer%20Iterationsrunde%20auskommt.)

### 12.7 [Sensitivity of different sequence comparison methods](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2286722/)

### 12.8 [Comparison of profiles and consensus sequences](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1874647/)

### 12.9 [DELTA-BLAST](https://bio.fandom.com/wiki/DELTA-BLAST)

# 13. Multiple sequence alignments and applications

* [https://www.sciencedirect.com/topics/medicine-and-dentistry/multiple-sequence-alignment#:~:text=Multiple%20sequence%20alignment%20(MSA)%20is,and%20analysed%20with%20computational%20algorithms.](https://www.sciencedirect.com/topics/medicine-and-dentistry/multiple-sequence-alignment#:~:text=Multiple%20sequence%20alignment%20(MSA)%20is,and%20analysed%20with%20computational%20algorithms.)

## 
